src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/casedocuments/ExpertEvidenceHandler.java

```diff
@@ -21,4 +23,19 @@ protected boolean canHandleDocument(UploadCaseDocument uploadCaseDocument) {
             && (caseDocumentType.equals(CaseDocumentType.VALUATION_REPORT)
             || caseDocumentType.equals(CaseDocumentType.EXPERT_EVIDENCE));
     }
+
+    @Override
+    protected DocumentCategory getDocumentCategoryFromDocumentType(CaseDocumentType caseDocumentType) {
+        switch (caseDocumentType) {
+            case VALUATION_REPORT -> {
```

Comment:
think you can remove some elements to something like case VALUATION_REPORT -> DocumentCategory.REPORTS; no need for {} and return clause to the whole switch block rather than each case